dvc/testing/benchmarks/cli/commands/test_update.py

```diff
@@ -3,7 +3,7 @@ def test_update(bench_dvc, tmp_dir, dvc, make_dataset):
     dataset = make_dataset(
         cache=False, files=True, dvcfile=False, commit=False, remote=False
     )
-    dvc.imp_url(str(dataset), "new")
+    bench_dvc("import-url", str(dataset), "new")
```

Comment:
We can't use the `dvc` fixture for benchmark tests. Using DVC 3.x will generate 3.x `.dvc` and `dvc.lock` files that cannot be read in older versions.

What we really need is an equivalent fixture for test setup that uses the venv DVC, but `bench_dvc` works for now